### ISSUE TITLE
definitions in frontmatter and glossary

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/wcag21#83]
WCAG 2.0 had definitions in the frontmatter for web page, programmatically determined, and accessibility supported, but also defined the first and third in the glossary. This makes it ambiguous which terms should be referenced in links, and resulted in duplicate definition errors from respec. I removed the defining instances of the duplicates but links to programmatically determined will still exceptionally go to the top whereas all other term links go to the glossary.
